test/dashboard-e2e/api/api-helpers.js

```diff
@@ -145,4 +158,8 @@ export class ApiHelpers {
             throw Error(`getLastExecutionNumber failed on "${request}" with: "${e}"`);
         }
     }
+
+    parseResponse(response) { // Workaround for empty response.body
+        return !this.cloudContext ? response.body : JSON.parse(response.text);
+    }
```

Comment:
Won't just `JSON.parse(response.text)` work alone?
```suggestion
    parseResponse(response) {
        // Cloud is missing the `content-type`, so there is no JSON response in `response.body`
        return JSON.parse(response.text);
    }
```

Reply:
I used it only for Cloud because OSS responses would need to also be parse then even if they are correct. But, it's fine to change it